README.md

```diff
@@ -1,3 +1,7 @@
+> [!NOTE]
+> This should be rendered obsolete by [the second generation](https://cloud.google.com/build/docs/automating-builds/github/build-repos-from-github?generation=2nd-gen)
+> of Google Cloud Build's GitHub integration.
+
 # gcb2gh
```

Comment:
Should be? Is it?

Reply:
It may be slightly less featureful, though. I don't remember it updating the stepwise status of the pipeline. I think it was just a quaternary thing: pending/waiting, running, succeeded, failed.